catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/low-bandwidth-map/low-bandwidth-map.hbs

```diff
@@ -0,0 +1,24 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<div class="low-bandwidth-confirmation">
+    <h3 align="center">
+    Low-bandwidth mode is enabled. Please confirm that you want this component to load despite potential bandwidth limitations. 
```

Comment:
Should this read 

>...despite potential bandwidth **implications**.

either that or remove` potential` because the bandwidth limitations are the current situation not the potential situation.

Reply:
Also while *we* know what the limitations/implications are, should we expand on this further for the more .. *inexperienced*.. user? Maybe explain that by continuing it will consume a large amount of the available connection resources to this web app, they may experience extended periods of waiting/timeouts for them and/or other users on the same network,  page may seem unresponsive.. etc etc?

Reply:
@rzwiefel How's the new wording?

Reply:
Looks good 👍